ansible_collections/arista/avd/roles/eos_designs/python_modules/connected_endpoints/ethernet_interfaces.py

```diff
@@ -98,6 +98,7 @@ def _get_ethernet_interface_cfg(self, adapter: dict, node_index: int, connected_
             "description": self.shared_utils.interface_descriptions.connected_endpoints_ethernet_interfaces(peer, peer_interface, adapter.get("description")),
             "speed": adapter.get("speed"),
             "shutdown": not adapter.get("enabled", True),
+            "sflow": self._get_adapter_sflow(adapter),
```

Comment:
You don't want it here, since this config applies to both port-channel members and regular ethernet interfaces. You have it in line 145 for the regular ethernet interfaces.